examples/end-to-end-examples/sec_10k_qa/mcli-yamls/06a_deploy_llm.yaml

```diff
@@ -1,35 +1,44 @@
 name: mpt-7b-finetuned
 
 # Deployment configuration
-gpu_num: 1  # Number of GPUs to use
-# cluster: r0z0 # Name of the cluster to use for this run
-gpu_type: a10 # Type of GPU to use.
+compute:
+  gpus: 1  # Number of GPUs to use
+  # cluster: r0z0 # Name of the cluster to use for this run
+  gpu_type: a10 # Type of GPU to use.
+  instance: oci.vm.gpu.a10.1
 replicas: 1 # Number of replicas to use for this run
 
 
 integrations:
 # Clone and install the examples repo so we can use the deployment helper from it
 - integration_type: git_repo
-  git_repo: mosaicml/examples # TODO: CHANGE BACK TO MAIN
+  git_repo: mosaicml/examples
```

Comment:
This should pin to a commit as well

Can use this one: `27230fa1c883c17a2a45a3762239ff7f5b0ef8ce`

Reply:
@dakinggg Note that this commit I suggested changes the API from what the old 7B used though

Reply:
I think the langchain integration is robust to all the variants of the API + Jeffreys backwards compatibility stuff, but @daya could you also test the langchain integration? Should be straightforward to instantiate the llm class and call it